src/backend/distributed/commands/grant.c

```diff
@@ -94,6 +94,8 @@ PreprocessGrantStmt(Node *node, const char *queryString,
 				ereport(ERROR, (errcode(ERRCODE_FEATURE_NOT_SUPPORTED),
 								errmsg("grant/revoke on column list is currently "
 									   "unsupported")));
+
+				return NIL;
```

Comment:
It is redundant to have return after ERROR, because we'd already return at that line